api.py

```diff
@@ -615,9 +615,26 @@ def get_article(
     """
     Fetch an individual article record by ID.
     """
+    source = {
+        "includes": [
+            "article_title",
+            "normalized_article_title",
+            "publication_date",
+            "indexed_date",
+            "language",
+            "full_language",
+            "canonical_domain",
+            "url",
+            "normalized_url",
+            "original_url",
+            "text_content",
+        ]
+    }
+    query = {"match": {"_id": id}}
 
     try:
-        hit = ES.get(index=collection.name, id=id)
+        res = ES.search(index=collection.name, source=source, query=query)
+        hit = res["hits"]["hits"][0]
```

Comment:
We should check `if res["hits"]["hits"]` first... Otherwise we risk `IndexError`.